user/articles/130-web-experience-management/03-content-management/04-content-sets/01-intro.markdown

```diff
@@ -0,0 +1,17 @@
+# Introduction to Content Sets
+
+A Content Set is exactly what it sounds like: a set of content items. In @product-ver@, an administrator can define a list of content, and then that list of content can be displayed somewhere. The way that the Content Set is displayed is determined by the method that is used to display it. For example, if the Content Set is being used by a smartwatch app, it could be displayed as a simple list of titles, and selecting a title would cause the full article to display on a connected mobile device. The same Content Set could be displayed in a web browser with the full content of each article.
```

Comment:
Maybe here it's worth mentioning the types of content that come out of the box with Liferay and are supported: web content articles, documents, images, videos, blog entries, ...
And also mention how it's not limited to them, since 3rd party developers can add their own types.